queries/pokemon_query.go

```diff
@@ -0,0 +1,39 @@
+package queries
```

Comment:
Same thing about package and file naming here, singular and no need for the suffix. Although here, you would probably like to specify the source of the information, in this case the csv file